frontend/packages/dev-console/src/components/gitops/GitOpsList.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import GitOpsListItem from './GitOpsListItem';
+import {
+  Stack,
+  StackItem,
+  Split,
+  SplitItem,
+  EmptyState,
+  EmptyStateVariant,
+  EmptyStateIcon,
+} from '@patternfly/react-core';
+import { getImageForIconClass } from '@console/internal/components/catalog/catalog-item-icon';
+import { GitOpsAppGroupData } from './gitops-types';
+import './GitOpsList.scss';
+
+interface GitOpsListProps {
+  appGroups: GitOpsAppGroupData[];
+  emptyStateMsg: string;
+}
+
+const GitOpsList: React.FC<GitOpsListProps> = ({ appGroups, emptyStateMsg }) => {
+  const emptyState = () => {
+    const gitopsImage = () => (
```

Comment:
This should be a static component defined outside of this component so that it is not created uniquely on each render.